cns/cns_util.go

```diff
@@ -123,6 +126,18 @@ func dropUnknownVolumeMetadataUpdateSpecElements(c *Client, updateSpecList []cns
 			updatedUpdateSpecList = append(updatedUpdateSpecList, updateSpec)
 		}
 		updateSpecList = updatedUpdateSpecList
+	} else if c.serviceClient.Version == ReleaseVSAN70 {
+		for _, updateSpec := range updateSpecList {
+			updateSpec.Metadata.ContainerCluster.ClusterDistribution = ""
+			var updatedContainerClusterArray []cnstypes.CnsContainerCluster
+			for _, containerCluster := range updateSpec.Metadata.ContainerClusterArray {
+				containerCluster.ClusterDistribution = ""
+				updatedContainerClusterArray = append(updatedContainerClusterArray, containerCluster)
+			}
+			updateSpec.Metadata.ContainerClusterArray = updatedContainerClusterArray
+			updatedUpdateSpecList = append(updatedUpdateSpecList, updateSpec)
+		}
+		updateSpecList = updatedUpdateSpecList
```

Comment:
Looks like we are updating the param and also returning it. Why is this required?

Reply:
We need this parameter because when the version is not VSAN67U3 nor VSAN70, then we wont change it.